EmbeddedScripts.Shared/ICodeRunner.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Threading.Tasks;
+
+namespace EmbeddedScripts.Shared
+{
+    public interface ICodeRunner
+    {
+        public Task RunAsync();
```

Comment:
nit: interface should not contains public/private modifiers - it's up to interface implementation